src/components/career-form/career-form.tsx

```diff
@@ -0,0 +1,191 @@
+import React, { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { ErrorMessage, TextArea } from '../form/controls';
+import { Grid, GridItem } from '../grid/grid';
+import axios from 'axios';
+import {
+  Actions,
+  Fieldset,
+  InputField,
+  ProgressBar,
+  SuccessMessage,
+} from './career-components';
+
+interface CareerFormProps {
+  recruiting_channel_id: string;
+  job_position_id: string;
+  access_token: string;
+  company_id: string;
+}
+
+type FormData = {
```

Comment:
Why a `type` and not an `interface`?

I usually only use `type` for primitive types like enums 
```
type Day = 'Monday' | 'Tuesday'
```

Reply:
I'm constantly torn between interfaces and types 
and I don't have a proper "best practice" yet. I have discussed with other colleagues a similar thought. It's not about primitives but about the type composition.

This boils down to: Use interface whenever possible and type in any other case `type` like for composition. I will change it here because I randomly picked one of the two ways here and I would like to align 👍